app/admin/events.rb

```diff
@@ -44,6 +44,13 @@
       f.input :eventbrite_url
       f.input :notes
     end
+
+    f.inputs 'Featured Projects' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.